Jenkinsfile

```diff
@@ -7,11 +12,17 @@ def buildApps(appList) {
   }
 }
 
-def runningOn(machine) {
-  println "Stage running on:"
-  println machine
+def buildDocs(){
+  sh 'pip install git+ssh://git@github.com/xmos/xmosdoc'
+  sh 'xmosdoc html latex'
+  script {
+    def doc_version = sh(script: "cat settings.yml | awk '/version:/ {print \$2}'", returnStdout: true).trim()
+    def zipFileName = "docs_xscope_fileio_v${doc_version}.zip"
+    zip zipFile: zipFileName, archive: true, dir: "doc/_build"
```

Comment:
Some jobs upload all the html sources so that they can be navigated from Jenkins without having to unzip. @BrennanGit is there a recommended/preferred thing we should do so we all repos are the same?

Reply:
I'd rather have them renderable by uploading them unzipped but a lot of developers who only use blue ocean find the flat list is fileld with html files that mean nothing on their own. We could encourage people to use classic mode for artifacts but I don't really see that happening.